install/install.md

```diff
@@ -163,8 +163,8 @@ bash /rapids/utils/stop-jupyter.sh
 
 <br/>
 
-### **pip (Experimental)**
-The package installer for python (pip) is currently in experimental mode, but available to try in both WSL2 or Ubuntu. See below for details.
+### **pip**
+The package installer for python (pip) is generally available as of RAPIDS release 23.04. See below for details.
```

Comment:
Do we need to state "generally"? Available seems sufficient, no need to draw attention to the fact that there was an EA period.

Reply:
Agree with this. It being there also somewhat dates the page after a while. Will change.